mcda/models/ProMCDA.py

```diff
@@ -0,0 +1,153 @@
+import sys
+import time
+import logging
+import pandas as pd
+from typing import Tuple
+
+from mcda.configuration.configuration_validator import extract_configuration_values, check_configuration_values
+
+log = logging.getLogger(__name__)
+
+FORMATTER: str = '%(levelname)s: %(asctime)s - %(name)s - %(message)s'
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG, format=FORMATTER)
+logger = logging.getLogger("ProMCDA")
+
+
+class ProMCDA:
+    def __init__(self, input_matrix:pd.DataFrame, polarity:Tuple[str], sensitivity:dict, robustness:dict,
```

Comment:
@kapil-agnihotri let's discuss it later as I feel it might be an important point. 
Shortly, my idea is to write a README for the user with all the specifications.